src/canmatrix/xls_common.py

```diff
@@ -84,17 +84,17 @@ def getSignal(db, sig, motorolaBitFormat):
     # startvalue of signal available
     if "GenSigStartValue" in db.signalDefines:
         if db.signalDefines["GenSigStartValue"].definition == "STRING":
-            frontArray.append(sig.attribute(db,"GenSigStartValue"))
+            frontArray.append(sig.attribute("GenSigStartValue"), db=db)
```

Comment:
Need to shift a paren outside the `db=db.  Presently it has `db` as a parameter to `.append()`.

Reply:
You are right, this must be fixed